src/Aspire.Hosting.AWS/CloudFormation/CloudFormationAWSConsoleUrlMapper.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.AWS.CloudFormation;
+
+internal static class CloudFormationAWSConsoleUrlMapper
+{
+    public static string MapCloudFormationUrl(string stackId)
+    {
+        var region = stackId.Split(":")[3];
```

Comment:
Can we make this more robust? I'd hate to see random exceptions because of format changes or parsing errors.

Reply:
Now that the client is used to resolve information I resolve this via a the client configuration instead.

Reply:
I added a new comment on how you can parse the stack arn using the SDK's `Arn` class.